### PR TITLE
Crns 121/message input

### PR DESCRIPTION
**Pull request by virdesai**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/Attachment/FileAttachment.js [hunk 1]
```diff
@@ -81,13 +81,13 @@ const FileAttachment = ({
...
       </FileContainer>
-      {attachment.actions?.length && (
+      {attachment.actions && attachment.actions.length ? (
         <AttachmentActions
           actionHandler={actionHandler}
           key={`key-actions-${attachment.id}`}
           {...attachment}
         />
-      )}
+      ) : null}
```
Comment (DanC5) [inline src/components/Attachment/FileAttachment.js:90]: do we need the ternary or can it be another && call above?

File: src/components/IconSquare.js [hunk 2]
```diff
@@ -7,30 +7,26 @@ const StyledTouchableOpacity = styled.TouchableOpacity`
...
-  else
-    return (
-      <StyledView>
-        <Image source={icon} style={{ height: 15, width: 15 }} />
-      </StyledView>
-    );
-};
+export const IconSquare = ({ icon, onPress }) =>
+  onPress ? (
+    <StyledTouchableOpacity onPress={onPress} testID={'icon-square'}>
```
Comment (DanC5) [inline src/components/IconSquare.js:22]: remove the brackets on these guys?

File: src/components/docs/Avatar.md [hunk 3]
```diff
@@ -8,48 +8,10 @@ const name = 'uthred';
 <Avatar image={image} name={name} size={35} />;
 ```
 
-A rounded Avatar
```
Comment (DanC5) [inline src/components/docs/Avatar.md:11]: do we no longer need this docs example?

Comment (virdesai) [reply src/components/docs/Avatar.md:11]: well there's no such thing as this as the avatar is already round...

Comment (DanC5) [reply src/components/docs/Avatar.md:11]: haha dope

**Pull request merged**